tests/test_s3.py

```diff
@@ -742,7 +757,7 @@ def test_bucket_in_other_region(self):
 
     @run_until_complete
     def test_bucket_in_other_region_using_http(self):
-        client = self.session.create_client(
+        client = self.create_client(
```

Comment:
Strang. Anyway it is fragile and something should be done about this.  Because asyncio does not allow such calls, here simple test:
```python
import asyncio

loop = asyncio.get_event_loop()

@asyncio.coroutine
def go(loop):
    yield from asyncio.sleep(1, loop=loop)
    loop.run_until_complete(asyncio.sleep(1, loop=loop))

loop.run_until_complete(go(loop))
```
will produce error:
```
  File "/usr/local/Cellar/python3/3.4.3/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/base_events.py", line 304, in run_until_complete
    self.run_forever()
  File "/usr/local/Cellar/python3/3.4.3/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/base_events.py", line 271, in run_forever
    raise RuntimeError('Event loop is running.')
RuntimeError: Event loop is running.
```